Apps/Sandcastle/gallery/Resolution Scaling.html

```diff
@@ -43,6 +43,15 @@
 function startup(Cesium) {
     'use strict';
 //Sandcastle_Begin
+// When browser recommended resolution is enabled, the viewer renders at
+// CSS pixel resolution (96 dppi). Otherwise, the viewer renders at the
+// screen's native resolution.
+
+// Resolution scale applies additional scaling on top.
```

Comment:
Maybe follow with an example.

> If the cesium container div is 500x500 CSS pixels, `window.devicePixelRatio` is 2.0, and `useBrowserRecommendedResolution` is false, the WebGL canvas will be 1000x1000 pixels. If `resolutionScale` is 0.25 then the resolution will be scaled down to 250x250 pixels.

Not a perfect example but something like that.